App/Config/app-config.ts

```diff
@@ -0,0 +1,7 @@
+import FS from 'react-native-fs'
+
+const config = {
+  textileRepoPath: `${FS.DocumentDirectoryPath}/textile-go`
```

Comment:
The sdk now requires that the host app pass in the repo path. More flexible that way, makes sense, etc etc.

Reply:
Just storing our value for repo path in a centralized location where any code can access it